teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    protein_data['accession'] = protein_data['target_id'].apply(lambda x: x.split('_')[0])
</pre><p>Add comment on the how and why of thesplit</p><ul><li>how? maybe simply by adding an example of before/after</li><li>why? we are selecting all target variants, WT and mutations?</li></ul><p> </p><p>Usually we add the why as Markdown text before the code, and the how (the technical bit) as code comment within the code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>